manifests/n/Nushell/Nushell/0.34.0/Nushell.Nushell.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Nushell.Nushell
+PackageVersion: 0.34.0
+PackageLocale: en-US
+Publisher: The Nu Project Contributors
+PackageName: nu
+License: MIT
+ShortDescription: A new type of shell
+Moniker: nushell
+ManifestType: defaultLocale
+ManifestVersion: 1.0.0

```

```suggestion
PackageIdentifier: Nushell.Nushell
PackageVersion: 0.34.0
PackageLocale: en-US
Publisher: The Nu Project Contributors
PackageName: nu
PackageUrl: https://github.com/nushell/nushell
License: MIT
LicenseUrl: https://github.com/nushell/nushell/blob/main/LICENSE
ShortDescription: A new type of shell
Moniker: nushell
Tags:
- shell
- rust
ManifestType: defaultLocale
ManifestVersion: 1.0.0
```
Just adding more metadata